### ISSUE TITLE
Some questions about the training

### ISSUE THREAD

**Issue by raywzy** [qibao77/CFSNet#2]
During training, only control parameters α =1 and α = 0 will be input into the network?

**Comment by qibao77** [comment]
Yes. In training step 1, we keep α_in= 0, while in training step 2, we keep α_in = 1.  I am sorry for the late reply.

**Comment by LaterBetterThanNever** [comment]
During testing, how can we choose the control parameters α to input into the network?

**Comment by qibao77** [comment]
It depends on your training settings. For example, for the JPEG image deblocking task in our experiment, we set quality factor as 10 in the first training stage and change it to 40 in the second training stage. Therefore, during testing, we change α_in from 0 to 1 with an interval of 0.1 to get different results of any input image with quality factor between 10 and 40. And we can choose the best α_in based on visual quality or PSNR.

**Comment by LaterBetterThanNever** [comment]
Thanks for replying!
